chapters/intro/introduction.tex

```diff
@@ -3,7 +3,12 @@ \chapter{Introduction}
 \section{Motivation}
 \section{Methodology}
 \subsection{Software Overview}
+I don't have a strong understanding of what might go in here
 \subsection{Implementation}
+or here.
+
+Good next steps are to start making an outline in these sections. 
+You can just use enumerate or itemize or something to make the detailed outline--the thing where each thing becomes its own paragraph in the long run :) 
```

Comment:
Just so it's documented here, I'm going to reiterate that I'm waiting to do the intro until last (or right before I write the conclusion), so these are really just placeholders.